src/core/time.d

```diff
@@ -4714,34 +4719,29 @@ double _abs(double val) @safe pure nothrow @nogc
   +/
 string numToString(long value) @safe pure nothrow
```

Comment:
LOL. Apparently, I didn't reload the page before commenting, and I didn't see either of the two comments that now precede mine. I think that it's better to do the numToString change in another PR, but it's certainly not the end of the world if it goes in this one.